### DISCUSSION TITLE
how to run c# console exe switch to use mono interpreter runtime?

### DISCUSSION THREAD

**Discussion by srxqds** [dotnet/runtime#60220] [Q&A]
can use `dotnet` command to set the mono interpreter runtime?

**Comment by ghost** [comment]
I couldn't figure out the best area label to add to this issue. If you have write-permissions please help me learn by adding exactly one [area label](https://github.com/dotnet/runtime/blob/master/docs/area-owners.md).

**Comment by danmoseley** [comment]
@SamMonoRT
